src/pages/Token/BundleModals/BundleModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { Modal } from '@unique-nft/ui-kit';
+
+import { BundleModalType } from './types';
+import { UnnestModal } from './UnnestModal';
+import { TransferToBundleModal } from './TransferToBundleModal';
+import { INestingToken } from 'components/BundleTree/types';
+
+export type TBundleModalProps = {
+  token?: INestingToken
+  parent?: INestingToken
+  bundleModalType: BundleModalType
+  onClose(): void
+  onFinish(): void
+  testid: string
+}
+
+export type TBundleModalBodyProps = {
+  token: INestingToken
+  parent: INestingToken
+  onFinish(): void
+  setIsClosable(value: boolean): void
+  testid: string
+}
+
+const BundleModal = ({ token, parent, onClose, onFinish, bundleModalType, testid }: TBundleModalProps) => {
```

Comment:
let's rename it to "BundleTrunsferModal". We can, potentially, have some sort of modal to see the internals of some bundle and this name will confuse us